shotgun_api3/shotgun.py

```diff
@@ -265,7 +265,9 @@ def __init__(self,
         :param connect: If True, connect to the server. Only used for testing.
 		
 		:param ca_certs: The path to the SSL certificate file. Useful for users
-		who would like to package their application into an executable.
+		who would like to package their application into an executable or for
+        corporate or internal CAs. Note that providing this parameter explicitly 
+        will override any value in the SHOTGUN_API_CACERTS environment variable. 
```

Comment:
We are referring to the `SHOTGUN_API_CACERTS` environment variable here as if it is a known thing, but it hasn't yet been mentioned in the docstring (as far as I can tell). Also, the previous doc string isn't great - "The path to the SSL certificate file...." - "**the** SSL certicate" seems to imply that this is required where it is optional and an expert setting. 

Suggested doc string to make this clearer:

```
:param ca_certs: Optional path to an external SSL certificate file. By default, the Shotgun API 
will use its own built-in certificates file, but if you are running a corporate or internal CA, 
it may be necessary to override these defaults. You can override it either by passing 
it in via this parameter or by setting the environment variable `SHOTGUN_API_CACERTS`.
In the case both are set, this parameter will take precedence.
```

KP, you are the master of grammar and words, so I am sure you will tweak and correct this further. Hope it's useful.